src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,70 @@
+import { SortType } from '../../types/SortType';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  sortField: SortType;
+  setSortField: (field: SortType) => void;
+  todos: Todo[];
+  clearCompleted: () => void;
+};
+
+export const Footer: React.FC<Props> = ({
```

Comment:
React.FC is not needed

https://github.com/typescript-cheatsheets/react?tab=readme-ov-file#section-2-getting-started